src/pages/profile.tsx

```diff
@@ -0,0 +1,135 @@
+
+export const Profile = () => {
+  const { page, setCurrentPage } = useContext(RouteContext)
+  const [user, setUser] = useState<null | { id: string, user_name: string, bio: string, created_at: string, linkedin_url: string, twitter_username: string, url: string, interests: string, open_issues: number }>(null)
+  const [userPR, setUserPR] = useState<null | { meta: {itemCount: number} }>(null)
+
+  useEffect(() => {
+    getUserData(page.props.userName).then((data) => { setUser(data)})
+    getUserPRData(page.props.userName).then((data) => { setUserPR(data)})
+  }, [])
+
+
+  return (
+    <div className="grid grid-cols-1 divide-y divider-y-center-2 min-w-[320px] text-white">
+      <header className='flex justify-between'>
+        <div className="flex items-center gap-2">
+          <button onClick={() => { setCurrentPage("home") }} className='rounded-full p-2 bg-slate-700 hover:bg-slate-700/50'>
+            <FaChevronLeft className='text-osOrange' />
+          </button>
+          <img src={OpenSaucedLogo} alt="OpenSauced logo" className='w-[100%]' />
+        </div>
+        <button 
+          title='Refresh user data'
+          className='hover:text-orange text-lg' 
+          onClick={() => {
+            getUserData(page.props.userName, true).then((data) => { setUser(data)})
+            getUserPRData(page.props.userName, true).then((data) => { setUserPR(data)})
+          }}>
+          <AiOutlineReload />
+        </button>
+      </header>
+      <main>
+        <div className='flex flex-col items-center gap-1 mb-4'>
+          <img
+            src={`https://github.com/${page.props.userName}.png`}
+            alt="profile image"
+            className='rounded-full w-14 aspect-square p-1 bg-slate-700' />
+          <p className='font-medium'>@{page.props.userName}</p>
+          {(user?.linkedin_url || user?.twitter_username) &&
+            <div className='social flex gap-0.5'>
+              {user?.linkedin_url &&
+                <a 
+                  target={'_blank'}
+                  href={user.linkedin_url} 
+                  title={user.linkedin_url} 
+                  className='rounded-sm border bg-slate-700 hover:bg-slate-700/50 hover:text-orange p-1'>
+                  <RiLinkedinFill className='text-lg' />
+                </a>
+              }
+              {user?.twitter_username &&
+                <a 
+                  target={'_blank'}
+                  href={`https://twitter.com/${user.twitter_username}`} 
+                  title={`https://twitter.com/${user.twitter_username}`} 
+                  className='rounded-sm border bg-slate-700 hover:bg-slate-700/50 hover:text-orange p-1'>
+                  <RiTwitterFill className='text-lg' />
+                </a>
+              }
+            </div>
+          }
+          {user?.bio && <p>This is my bio</p>}
+          {user?.url &&
+            <a target={'_blank'} href={user.url} className='flex text-orange items-center gap-0.5'>
```

Comment:
```suggestion
            <a target={'_blank'} href={user.blog_url} className='flex text-orange items-center gap-0.5'>
```
url for opensauced goes to github, which we are already there. Instead we should use blog_url to view their website.